addons/fix-uploaded-svgs/userscript.js

```diff
@@ -1,4 +1,75 @@
 export default async function ({ addon, console }) {
+  // Fixes "x", "y", and "dominant-baseline" being ignored by Scratch, "span" causing offset
+  function toEditorSVG(element) {
```

Comment:
Maybe this function could be named fixEditorSVG and include previous fix too?

Reply:
Sure! Also, the previous fix should still be there, although it is outside "toEditorSVG"... would you like me to incorporate it into the function?

Reply:
Sure, so the actual SVG only needs to be modified once.